### PR TITLE
Tag page redesign

### PR DESCRIPTION
**Pull request by katjuell**
### What should this PR do?
Resolves [DEVED-230](https://sourcegraph.atlassian.net/browse/DEVED-230) by redesigning the tag landing page to be more dynamic.

### Why are we making this change?
- We want the site and its main pages to be visually engaging for users, while also surfacing relevant content in interesting ways.

### What are the acceptance criteria? 
- Tag pages designs should look reasonable and visually appealing.
- Featured content is display prominently in new design (and listed in record list).
- Record list tabs are differentiated by genre (tutorial/video).
- It is possible to tab between record list tabs.
- Active tabs are highlighted.
- Load more functionality and logic works as it currently does on prod, with the same parameters.

### How should this PR be tested?
- Check out the branch, go to a tag page, and test the above criteria.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

File: components/atoms/Column/ColumnStyles.ts [hunk 1]
```diff
@@ -23,13 +23,13 @@ export const StyledColumn = styled.div`
 
     &.flex-small {
         flex: 0 0 auto;
-        width: 25%;
+        width: 20%;
```
Comment (katjuell) [inline components/atoms/Column/ColumnStyles.ts:26]: Parity with #366 until that is merged.

**Comment by katjuell** [comment]
No records:

Screen Shot 2021-11-11 at 9 53 09 AM

Desktop:
Screen Shot 2021-11-11 at 9 53 30 AM

Active tab:
Screen Shot 2021-11-11 at 9 54 02 AM

IPad:
Screen Shot 2021-11-11 at 10 27 08 AM

Mobile:
Screen Shot 2021-11-11 at 10 27 42 AM

Screen Shot 2021-11-11 at 10 27 58 AM

**Comment by katjuell** [comment]
@ltagliaferri marking this as ready for review to get your initial thoughts on the design.

**Review by ltagliaferri — COMMENTED** [review]
I am liking this, I really like the `Tutorials` / `Videos` toggle at the bottom of the page.

Some thoughts on design only right now, and will review again as this continues to progress.

I like what is happening with the three cards at the top of the tag pages. Right now, I prefer how it looks better on the `Open Source` page as the two cards on the right take up the space more. On `Sourcegraph` it looks like they are not taking up enough space. Could you force the size of them to line up vertically with the bigger card on the left? I understand for long titles this may not be an ideal scenario, but maybe the tags can be killed on the smaller cards?

Screen Shot 2021-11-11 at 11 37 28 PM 
Screen Shot 2021-11-11 at 11 36 09 PM

Regarding the `Tutorials` / `Videos` at the bottom, can you add logic so that when there are no records the `Load more` button does not appear?

Screen Shot 2021-11-11 at 11 36 19 PM

I noticed that the `GitHub` tag displays as `Github` and modifying the markdown file with it did not change it locally (it is only one tutorial). Is there a preferred way of casing between tags and tag index pages?

**Review by ltagliaferri — APPROVED** [review]
Looks good to me, a lot of good additions! Everything is working as expected and discussed.

For a future PR, would it be a big lift to float the text towards the center of the cards on the right?

Screen Shot 2021-11-15 at 5 53 04 PM

**Comment by katjuell** [comment]
@ltagliaferri all set with that request — let me know if this isn't what you were thinking
Screen Shot 2021-11-16 at 4 01 15 PM
!

**Comment by ltagliaferri** [comment]
@katjuell — looks perfect, thank you!

**Pull request merged**